trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/ColumnIndices.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+/**
+ * Provides setters focused on configuring a validator to look at particular
+ * column within a row of a provided file
+ */
+trait ColumnIndices {
+
+  /**
+   * Sets the array of indices for which the validator to do its checks on.
+   *
+   * @param array $indices
+   *   An array of integers or keys which correspond to columns in a row of
+   *   delimited values that the validator instance should act on
+   * @return void
+   *
+   * @throws \Exception
+   *  - If $indices array is empty
+   *  - If $indices array is multi-dimensional or contains objects
+   */
+  public function setIndices(array $indices) {
+
+    // Make sure we don't have an empty array
+    if(count($indices) === 0) {
+      throw new \Exception('The ColumnIndices Trait requires a non-empty array of indices.');
+    }
+
+    // Check if we have a multidimentsional array or array of objects
+    foreach ($indices as $index) {
+      if (is_array($index) || is_object($index)) {
+        throw new \Exception('The ColumnIndices Trait requires a one-dimensional array only.');
+      }
+    }
+
+    // Set the indices array
+    $this->context['indices'] = $indices;
+  }
+
+  /**
+   * Returns an array of integers or keys which correspond to columns in a row
+   * of delimited values that the validator instance should act on.
```

Comment:
What about this wording instead to be consistent with what I suggested above?

```suggestion
   * Returns the indices of the columns this validator should act on.
   *
   * Specifically, in this context, an index is the key of the column the  
   * validator should check. In the case of lists, this index/key is a sequential  
   * integer created by PHP and in the case of associative arrays, this index/key  
   * will likely be a meaningful string (such as names of column headers).  
```